mantidproject/mantid
# Use dQQ to stitch workspaces in Reflectometry GUI

Scientists want to use the value in column `dQ/Q` to stitch workspaces that belong to the same group. They should still be able to use the `Stitch1DMany` text box in `Settings` tab.

**To test:**

Open MantidPlot -> Interfaces -> Reflectometry -> ISIS Reflectometry (Polref). Open table `INTER_NR_test2.tbl` from the ISIS sample dataset.
1. Select the first group and click `Process`, you should not get any error.
2. Remove values from column `dQ/Q` and run `Process` again, an error message should be shown.
3. Go to `Settings` tab and enter `Params=-0.04`, click on process, you should not get any errors.
4. Select the first group and click on the icon to plot group, you should see the same plot in cases 1 and 
5. Select the first group and click on the icon to plot rows, you should see the same plot in cases 1 and 3.

Fixes #18100.

Release notes were updated.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
